libs/execution_context/ebpf_maps.c

```diff
@@ -454,7 +454,7 @@ _create_per_cpu_hash_map(_In_ const ebpf_map_definition_t* map_definition)
     uint32_t cpu_count;
     uint32_t index;
     size_t per_cpu_size = 0;
-    ebpf_core_map_per_cpu_t* per_cpu = NULL;
+    ebpf_core_per_cpu_data_t* per_cpu = NULL;
```

Comment:
nit: variable name should be consistent with data type
```suggestion
    ebpf_core_per_cpu_data_t* per_cpu_data = NULL;
```